fennel/datasets/datasets.py

```diff
@@ -441,6 +444,30 @@ def dsschema(self):
         return self.node.dsschema()
 
 
+class Changelog(_Node):
+    def __init__(self, node: _Node, delete_column: str):
+        super().__init__()
+        self.node = node
+        self.delete_column = delete_column
+        self.node.out_edges.append(self)
+
+    def signature(self):
+        return fhash(self.node.signature(), self.delete_column)
+
+    def dsschema(self):
+        input_schema = self.node.dsschema()
+        # Remove all keys from the schema and make then values
+        val_fields = copy.deepcopy(input_schema.keys)
+        values = input_schema.values
+        val_fields.update(values)
+        val_fields.update({self.delete_column: pd.BooleanDtype})
```

Comment:
Do we store pandas type in fields or python one? If python then we should use bool instead of pd.BooleanDtype

Reply:
its pandas from what I see